core/spec/models/spree/page_sections/featured_posts_spec.rb

```
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe Spree::PageSections::FeaturedTaxon do
```

RuboCop (1.75.5)

[convention] Spec path should end with `spree/page_sections/featured_taxon*_spec.rb`.

(RSpec/SpecFilePathFormat)